PCA_shear.py

```diff
@@ -0,0 +1,155 @@
+import sys
```

Comment:
It's hard to understand. I will come back if there're some equations to compare this to.